gulpfile.js

```diff
@@ -7,11 +8,13 @@ var harpConfig  = require('./harp.json');
 var jimp        = require('gulp-jimp');
 var concat      = require('gulp-concat');
 var uglify      = require('gulp-uglify');
+var runSequence = require('run-sequence');
+
```

Comment:
If we ran this through JSLint, I'm pretty sure we'd get a "too many var declarations", we only need one:
```
var gulp = require('gulp'),
    browserSync = require('browser-sync'),
[...]
```
Also, introducing extra spaces between var name and "=" operator is not standard practice and IMHO doesn't make for an easier read.